Source/WebCore/rendering/StyledMarkedText.cpp

```diff
@@ -147,10 +147,10 @@ static Vector<StyledMarkedText> coalesceAdjacentWithSameRanges(Vector<StyledMark
             if (previousStyledMarkedText.priority <= it->priority) {
                 previousStyledMarkedText.priority = it->priority;
                 // If highlight, take textDecorationStyles of latest StyledMarkedText.
-                // FIXME: Check for taking textDecorationStyles needs to be changed to accommodate other MarkedText type
+                // FIXME: Check for taking textDecorationStyles needs to be changed to accommodate other MarkedText type.
```

Comment:
Is this a "should be on bugzilla" type FIXME?

Reply:
what do you mean?

Reply:
got it!